torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -191,6 +199,20 @@ bool isSupported(Node* node) {
       }
     }
 
+    // Value is only supported if operands are floats.
+    if (node->isMemberOf(float_only_operator_set)) {
+      for (const Value* v : node->inputs()) {
+        if (auto const& tt = v->type()->cast<TensorType>()) {
+          auto const& st = tt->scalarType();
+          if (!st || !isFloatingType(*st)) {
+            return false;
+          }
+        } else if (!v->type()->cast<FloatType>()) {
```

Comment:
You could also allow constant non-zero ints:

`(v->type()->cast<IntType>() && constant_as<int>(v).value_or(0) != 0)`